recipes/dvc/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "dvc" %}
+{% set version = "0.53.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 31d92fc583351664ca48df146e115c66ab41070520bb02ef35e14a9dec156ea5
+
+build:
+  number: 0
+  skip: True  # [py==37]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ply>=3.9
```

Please leave a space between package name and version spec for all requirements.
```suggestion
    - ply >=3.9
```

---

@xhochy done! Thanks